.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/Auterion/android_openssl
 [submodule "libs/qmlglsink/gst-plugins-good"]
 	path = libs/qmlglsink/gst-plugins-good
-	url = https://github.com/mavlink/gst-plugins-good.git
+	url = https://github.com/fredowski/gst-plugins-good.git
```

Comment:
You should remove this commit once your changes are merged upstream.

Reply:
That commit must not be merged to master. This PR has to wait on https://github.com/mavlink/gst-plugins-good/pull/2 . That are the backported bugfixes for gst-plugins.